src/__tests__/validate-selector.test.ts

```diff
@@ -29,6 +29,17 @@ describe("validateSelector", () => {
     expect(validate("@SUPPORTS (-webkit-appearance:none) {}")).toBe(false);
   });
 
+  it("is valid when the only at-rule present is @media", () => {
+    expect(validate("@media")).toBe(true);
+    expect(validate("@MEDIA")).toBe(true);
+  });
+
+  it("@ command is present, but any other @ command is not allowed", () => {
```

Comment:
Missed this test title in previous review:

```suggestion
  it("is not valid when disallowed at-rules are present", () => {
```